gazebo_plugins/include/gazebo_plugins/gazebo_ros_wheel_slip.hpp

```diff
@@ -77,28 +77,29 @@ class GazeboRosWheelSlipPrivate;
 ///      If not specified, these will default to 0.0.
 ///      Any negative values will ignored and set to 0.0.
 ///
-///  2. ROS parameters for each wheel, e.g :
+///  2. ROS parameters for all wheels, e.g :
+///      <ros>
+///         <parameter name="slip_compliance_unitless_lateral" type="double">0.1</parameter>
+///         <parameter name="slip_compliance_unitless_longitudinal" type="double">0.2</parameter>
+///      </ros>
+///
+///     If not specified, these will default to the last values set in SDF tags.
+///     If these are specified, they override SDF parameters for each wheel.
+///
+///  3. ROS parameters for each wheel, e.g :
 ///       <ros>
 ///         <parameter name="slip_compliance_unitless_lateral/wheel_front" type="double">0.1
 ///           </parameter>
 ///         <parameter name="slip_compliance_unitless_longitudinal/wheel_front" type="double">0.2
 ///           </parameter>
 ///       </ros>
 ///
-///      If not specified, these default to SDF parameters set for each wheel.
-///      If specified, they will override the values set as SDF parameters.
-///
-///  3. ROS parameters for all wheels, e.g :
-///      <ros>
-///         <parameter name="slip_compliance_unitless_lateral" type="double">0.1</parameter>
-///         <parameter name="slip_compliance_unitless_longitudinal" type="double">0.2</parameter>
-///      </ros>
-///
-///     If not specified, these will default to the last values set in SDF tags.
-///     If these are specified, they override both SDF and the ROS parameters for each wheel.
+///      If not specified, these default to the value of the ROS parameter for all wheels.
+///      If specified, they will override values set as SDF parameters or ROS parameters for all
```

Comment:
* for each wheel

Reply:
I'm trying to communicate that these parameters for individual wheels will override parameters that apply to all wheels. The language is a bit confusing, maybe we can clarify terms being used.

Reply:
I've attempted to clarify the documentation, PTAL: [aa06988](https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1365/commits/aa0698835c18ff71a70226e86bc52f4503113f78)